src/0_utils.js

```diff
@@ -51,7 +51,7 @@ utils.readStore = function() {
 		return JSON.parse(sessionStorage.getItem('branch_session')) || {};
 	}
 	catch (e) {
-		return {};
+		return {}; //TODO: Look into why we dont return the error
```

Comment:
Hrm, the likelihood that there's an error here is incredibly small - someone would have to screw with the sessionstorage. which is possible, but in that cas, we can just cancel it and do a new open. Make sense?